scripts/zones/AlTaieu/mobs/Ulxzomit.lua

```diff
@@ -2,43 +2,41 @@
 -- Area: Al'Taieu
 --  Mob: Ul'xzomit
 -----------------------------------
+mixins = { require('scripts/mixins/follow') }
+local ID = zones[xi.zone.ALTAIEU]
+-----------------------------------
 local entity = {}
 
-local followOptions = { forceRepathInterval = 1 }
+local leaders =
+{
+    [ID.mob.ULXZOMIT_OFFSET[1]] = 2,
```

Comment:
If these are all max distances, and then we set a group max distance of 2 in the call to assignLeaderMod, isn't all of this redundant?

Reply:
The `leaders` table is read from checking for which mobs are the leaders. I was able to trim this down to the more compact code.
```lua
local leaders = {}
for i = 0, 14 do
    leaders[ID.mob.ULXZOMIT_OFFSET[1 + i * 3]] = 2
end
```